### PR TITLE
Redesign Supercharge screen

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Simulator Screen Shot - iPhone SE (2nd generation) - 2022-02-18 at 15 12 06
Simulator Screen Shot - iPhone SE (2nd generation) - 2022-02-18 at 15 13 38
Simulator Screen Shot - iPhone SE (2nd generation) - 2022-02-18 at 15 13 53

### Other changes

Removed some unused feature flags.

### Tested

Manually and with unit tests.

### How others should test

- Open the screen while not verified, check that there's a bullet point prompting you to verify.
- Open the screen with no balance, check that there's a bullet point prompting you to add a balance.
- Open the screen with a verified number and more than 10 cUSD, check that you see the 'Supercharging' text.

### Related issues

- Part of #2017

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/locales/base/translation.json [hunk 1]
```diff
@@ -160,22 +160,18 @@
...
+  "superchargeDescription": "Get {{apy}}% annual rewards on your {{token}} balance. Here's how to start:",
+  "superchargeConnectNumber": "Connect your phone number",
+  "superchargeMinimumBalance": "Add a minimum balance of {{amount}} {{token}}",
+  "superchargeDisclaimer": "Max-out rewards by adding {{amount}} {{token}}",
+  "superchargeTokenDetailsDialog": {
+    "title": "Which assets can I Supercharge?",
+    "body": "With Supercharge, you get weekly rewards on your highest Celo stablecoin balance (cEUR, cUSD, cREAL, etc).",
+    "dismiss": "Dismiss"
   },
-  "conclusion": "It’s that simple. Supercharge your crypto today!",
```
Comment (jeanregisser) [inline packages/mobile/locales/base/translation.json:178]: Should we keep some of these to update the wording on older Valora versions?

File: packages/mobile/src/components/Pill.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+interface Props {
+  text: string
+  icon?: React.ReactNode
+  onPress: () => void
+  testID?: string
+}
+
+function Pill({ text, icon, onPress, testID }: Props) {
+  return (
+    <TouchableOpacity style={styles.container} onPress={onPress} testID={testID}>
```
Comment (jeanregisser) [inline packages/mobile/src/components/Pill.tsx:15]: Can we use `Touchable` instead?

File: packages/mobile/src/redux/migrations.ts [hunk 3]
```diff
@@ -423,4 +424,12 @@ export const migrations = {
...
     },
   }),
+  34: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      superchargeApy: REMOTE_CONFIG_VALUES_DEFAULTS.superchargeApy,
+      superchargeTokens: [],
+    },
+  }),
```
Comment (jeanregisser) [inline packages/mobile/src/redux/migrations.ts:434]: Can we remove the removed `rewardsX` keys?

File: packages/mobile/test/schemas.ts [hunk 4]
```diff
@@ -958,6 +958,25 @@ export const v33Schema = {
...
 }
 
+export const v34Schema = {
+  ...v33Schema,
+  _persist: {
+    ...v33Schema._persist,
+    version: 34,
+  },
+  app: {
+    ...v33Schema.app,
```
Comment (jeanregisser) [inline packages/mobile/test/schemas.ts:968]: We can remove the `rewardsX` keys too here.

File: packages/mobile/ios/Podfile.lock [hunk 5]
```diff
@@ -1068,7 +1068,7 @@ SPEC CHECKSUMS:
   Flipper-RSocket: d9d9ade67cbecf6ac10730304bf5607266dd2541
   FlipperKit: d8d346844eca5d9120c17d441a2f38596e8ed2b9
   fmt: ff9d55029c625d3757ed641535fd4a75fedc7ce9
-  glog: 5337263514dd6f09803962437687240c5dc39aa4
+  glog: 40a13f7840415b9a77023fbcae0f1e6f43192af3
```
Comment (jeanregisser) [inline packages/mobile/ios/Podfile.lock:1071]: Once `https://github.com/valora-inc/wallet/pull/2035` lands this shouldn't be necessary anymore.
I had to `rm -rf Pods` locally to get the right `glog` checksum which matched CI.

**Review by jeanregisser — APPROVED** [review]
Looking great! 👍 

Already approved with some minor comments.

One question about releasing this, I see it already mentions the claim part, but that part is still being worked on.
When do we plan to release this change? Given next cut is Wednesday 23rd.

**Comment by gnardini** [comment]
>One question about releasing this, I see it already mentions the claim part, but that part is still being worked on.
When do we plan to release this change? Given next cut is Wednesday 23rd.

Working on it, should be able to open a PR today but we are indeed very close to the deadline

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified the above task on Latest Android Internal Release Build & Test flight Release Build V1.27.0 and updated status in Test rails. 

**Test rail links:**
**Android 10:** https://valoraapp.testrail.io/index.php?/tests/view/12925

**iOS 14:** https://valoraapp.testrail.io/index.php?/tests/view/12987

**iOS 15:** https://valoraapp.testrail.io/index.php?/tests/view/13018

**Devices:** iPhone 13 mini(15.1.1), iPhone 12 (14.7.1), Samsung Galaxy M40 (10.0)
cc: @gnardini 
Thanks..!!

**Pull request merged**